BMS_Display.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include "BMS_Types.h"
+
+void DisplayBatteryFineStatus()
+{
+    Language_u8 == 1  ? printf("Battery Fine!\n") :  printf("Batterie gut !\n");
+}
+void DisplayParameterLLErrorStatus(char * BatterParameterName_p ,float ParamLimit_f)
+{
+   ( Language_u8 == 1  ? printf("Alert: %s  is less than %f !\n",BatterParameterName_p,ParamLimit_f) :
```

Comment:
The complexity of this function increases with number of languages supported. Any way to prevent that?